File: typescript_discord_bot_startup_sl8s/app/src/named_utility/factory_model_wrapper_repository_utility.ts
import ExampleWrapperRepository from "../model_wrapper_repository/example_wrapper_repository/example_wrapper_repository";

class FactoryModelWrapperRepositoryUtility {
    private constructor() {
    }

    public static get getExampleWrapperRepository(): ExampleWrapperRepository {
        return new ExampleWrapperRepository();
    }
}

export default FactoryModelWrapperRepositoryUtility;

File: dart_web_startup_sl8s/app/lib/named_utility/factory_model_wrapper_repository_utility.dart
import 'package:meta/meta.dart';
import 'package:dart_web_startup_sl8s/model_wrapper_repository/example_wrapper_repository/example_wrapper_repository.dart';

@immutable
final class FactoryModelWrapperRepositoryUtility {
  const FactoryModelWrapperRepositoryUtility._();

  static ExampleWrapperRepository get getExampleWrapperRepository {
    return const ExampleWrapperRepository();
  }
}